packages/atchops/tests/CMakeLists.txt

```diff
@@ -20,3 +20,4 @@ foreach(file ${files})
   target_link_libraries(${filename} PRIVATE atchops atlogger)
   add_test(NAME ${filename} COMMAND $<TARGET_FILE:${filename}>)
 endforeach()
+enable_testing()
```

Comment:
I snuck in some adjustments to enable_testing, so we don't build and run mbedtls & cjson tests all the time.